client/src/views/devices/ExportRecoveryDevicePage.vue

```diff
@@ -0,0 +1,208 @@
+<!-- Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS -->
+
+<template>
+  <ion-page>
+    <ion-content :fullscreen="true">
+      <div class="container">
+        <div class="recovery-container">
+          <div v-show="state === ExportDevicePageState.Start">
+            <ms-informative-text>
+              {{ $t('ExportRecoveryDevicePage.subtitles.newPassword') }}
+            </ms-informative-text>
+            <ms-informative-text>
+              {{ $t('ExportRecoveryDevicePage.subtitles.twoFilesToKeep') }}
+            </ms-informative-text>
+
+            <div>
+              <div class="block">
+                {{ $t('ExportRecoveryDevicePage.titles.recoveryFile') }}
+              </div>
+              <div class="block">
+                {{ $t('ExportRecoveryDevicePage.titles.recoveryKey') }}
+              </div>
+            </div>
+            <ion-button
+              @click="exportDevice()"
+            >
+              {{ $t('ExportRecoveryDevicePage.actions.understand') }}
+            </ion-button>
+          </div>
+          <div v-if="state === ExportDevicePageState.Download">
+            <ms-informative-text>
+              {{ $t('ExportRecoveryDevicePage.subtitles.keepFilesSeparate') }}
+            </ms-informative-text>
+
+            <div>
+              <div class="block">
+                {{ $t('ExportRecoveryDevicePage.titles.recoveryFile') }}
+                <div v-if="!recoveryFileDownloaded">
+                  <ion-button
+                    @click="downloadRecoveryFile()"
+                  >
+                    {{ $t('ExportRecoveryDevicePage.actions.download') }}
+                  </ion-button>
+                </div>
+                <div v-else>
+                  <ion-icon
+                    :icon="checkmarkCircle"
+                    class="checked"
+                  />
+                  {{ $t('ExportRecoveryDevicePage.subtitles.fileDownloaded') }}
+                </div>
+              </div>
+              <div class="block">
+                {{ $t('ExportRecoveryDevicePage.titles.recoveryKey') }}
+                <div v-if="!recoveryKeyDownloaded">
```

Comment:
`v-show` are probably better in those cases, mainly because it's easier to test.

`v-if` can be useful in cases where you have an undefined object for example:
```typescript
const obj: Ref<object | null> = ref(null);
```
```vue
<div v-if="obj">
  <!-- You have a guarantee that within this div tag, obj will never be null -->
  {{ obj.attribute }}
</div>
```

For testing, it's easier to have `cy.get('element').should('not.be.visible')` rather than testing whether the tag exists or not.